packages/harvester/contracts/test/MockHarvester.sol

```diff
@@ -5,6 +5,9 @@ import "../base/KeeperCompatibleHarvester.sol";
 
 contract MockHarvester is KeeperCompatibleHarvester {
     uint256 public vaultCount;
+    bool public canHarvest = true;
+    bool public shouldHarvestVault = true;
+    bool public didHarvest = true;
```

Comment:
Please set these in the constructor like `vaultCount`.

Reply:
Ok, I'll do that. Here is just about a style right? Because it's a mock contract? But that will cost more gass.